docs/releaseNotes.yml

```diff
@@ -32,6 +32,16 @@
 
 changelog: https://github.com/emissary-ingress/emissary/blob/$branch$/CHANGELOG.md
 items:
+  - version: 3.1.0
+    date: 'TBD'
+    notes:
+      - title: Add support for secrets sync
+        type: feature
+        body: >-
+          Adding an option to the directive service in the agent to synchronize secrets. The objective is to allow
+          a third party product to automate the creation of CRDs depending on a secret to work
```

Comment:
suggestions: 
- use "command" instead of "option" since that is what it is called.
- "synchronize" gives the impression that it is two way. Maybe use "manage" secrets.
- remove "The objective" to more direct language like "This allows"

Maybe something like so:

> Adds a new directive command to the $productName$ Agent service that allows it to manage secrets. This allows a third party product to manage $productName$ CRD's that depend upon a secret.